tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/Package/RunTestsTool.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.ComponentModel;
+using ModelContextProtocol.Server;
+using Azure.Sdk.Tools.Cli.Commands;
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Services;
+using Azure.Sdk.Tools.Cli.Services.Tests;
+
+namespace Azure.Sdk.Tools.Cli.Tools.Package
+{
+    /// <summary>
+    /// This tool runs validation checks for SDK packages based on the specified check type.
+    /// </summary>
+    [Description("Run tests for SDK packages")]
+    [McpServerToolType]
+    public class RunTestsTool(
+        ILogger<RunTestsTool> logger,
+        ILanguageSpecificResolver languageServiceResolver
+    ) : MCPTool
+    {
+        public override CommandGroup[] CommandHierarchy { get; set; } = [SharedCommandGroups.Package];
+
+        private const string RunTestsCommandName = "run-tests";
+
+        protected override Command GetCommand()
+        {
+            var command = new Command(RunTestsCommandName, "Run tests for SDK packages");
+            // Add the package path option to the parent command so it can be used without subcommands
+            command.AddOption(SharedOptions.PackagePath);
+            command.SetHandler(ctx => HandleCommand(ctx, ctx.GetCancellationToken()));
+
+            return command;
+        }
```

Comment:
This area has been simplified a bit under the hood.

```suggestion
        protected override Command GetCommand() => [
            new(RunTestsCommandName, "Run tests for SDK packages") {
                SharedOptions.PackagePath
            }
        ];
```